src/modules/webpack-publish.ts

```diff
@@ -20,7 +22,24 @@ const WebpackPublish = {
     getOptions(): OptionStringBuilder {
         return new OptionStringBuilder("publish", "p");
     },
-    run(): boolean {
+    restore(options: WebpackRestoreOptions): void {
+        const ci = options.ci ?? false;
+        const clean = !options.skipClean ?? true;
```

Comment:
Looking at this again - it might read better with the null coalesce expression inside parens - `!(options.skipClean ?? true)`. Prettier might overrule that and take them out though, not sure. Not a big deal either way.